apisix/plugins/opa/helper.lua

```diff
@@ -45,16 +52,136 @@ local function build_http_request(conf, ctx)
 end
 
 
-function _M.build_opa_input(conf, ctx, subsystem)
-    local request = build_http_request(conf, ctx)
+local function _build_http_route(conf, ctx)
+    local route_id = ctx.route_id
+    local routes = get_routes()
```

Comment:
We can use https://github.com/apache/apisix/blob/3c0b374808903388f5facc081dbdb2db672c7c69/apisix/init.lua#L388

Reply:
changed